src/csvcubed/models/validatedmodel.py

```diff
@@ -0,0 +1,42 @@
+# this class will be class attributes and returns an array of errors
```

Comment:
Can you make sure you use a docstring in [the appropriate format](https://github.com/GSS-Cogs/csvcubed/blob/main/docs/developer.md#api-documentation) here?